src/python_scripts/afni_python/db_mod.py

```diff
@@ -11958,7 +11958,7 @@ def show_help_examples():
             Please see 'auto_warp.py -help' for more information.
             See also -tlrc_opts_at, -anat_uniform_method.
 
-        -tlrc_NL_warped_dsets ANAT WARP.1D NL_WARP: import auto_warp.py output
+        -tlrc_NL_warped_dsets ANAT WARP.1D NL_WARP: from afni_python import auto_warp.py output
```

Comment:
need to fix this

Reply:
You do not need to go through all of this.